.rubocop.yml

```diff
@@ -16,6 +16,7 @@ Metrics/ClassLength:
   Exclude:
     - 'app/models/message.rb'
     - 'app/models/conversation.rb'
+    - 'enterprise/app/services/captain/llm/system_prompts_service.rb'
```

Comment:
lets not do this, we have rubocop to ensure that we breakdown the classes into readable sub services. the ignore list is only for the exceptions which were present when we introduced this, the idea is to remove the files from here going forward  and try best not to add more here. !!!

Reply:
note looked at the file and seems like it needs as overall rewam, lets do inline attribution in that case like how the other methods have handled it.